client/my-sites/plans/jetpack-plan-price/index.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' );
+
+module.exports = React.createClass( {
+	displayName: 'JetpackPlanPrice',
+
+	render: function() {
+		return (
+			<div className="plan-price">
```

Comment:
We should use the proper component class name here (`jetpack-plan-price`). See the docs for more details: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md